### PR TITLE
Test Errno by value rather than description

### PR DESCRIPTION
**Pull request by docelic**
This patch tests errno by value rather than by textual description of the error.

### CODE REVIEW

File: src/amber/server/server.cr [hunk 1]
```diff
@@ -74,7 +74,7 @@ module Amber
           server.listen(settings.port_reuse)
           exit
         rescue e : Errno
-          if e.message == "accept: Too many open files"
+          if e.errno == Errno::E2BIG
```
Comment (elorest) [inline src/amber/server/server.cr]: Is E2BIG equivalent to "accept: Too many open files"?

**Comment by docelic** [comment]
@elorest please check if this looks reasonable since it is a change to your original code. Thanks.

**Comment by elorest** [comment]
@docelic please review my comment above and advise before merging.

File: src/amber/server/server.cr (hunk 1 above)
Comment (docelic) [reply src/amber/server/server.cr]: Yes, I looked it up in Crystal sources.

**Comment by elorest** [comment]
Ok could probably be this same PR when you add that.

**Pull request closed without merging**